bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,465 @@
+	</channel-type>
+
+	<channel-type id="ip-address" advanced="true">
+		<item-type>String</item-type>
+		<label>IP Address</label>
+		<description>IP address of the client</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="led">
+		<item-type>Switch</item-type>
+		<label>Led Activated</label>
+		<description>Led indicator status</description>
+		<category>Switch</category>
+	</channel-type>
+
+	<channel-type id="gain">
+		<item-type>Dimmer</item-type>
+		<label>Gain</label>
+	</channel-type>
+
+	<channel-type id="connection-status">
+		<item-type>String</item-type>
+		<label>Connection</label>
+		<description>Connection Status</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="hardware-status">
+		<item-type>String</item-type>
+		<label>Hardware status</label>
+		<description>Hardware status of the phone line</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="telephony-service">
+		<item-type>String</item-type>
+		<label>Telephony Service</label>
+		<description>Status of the telephony service</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="host-count">
+		<item-type>Number</item-type>
+		<label>Host Count</label>
+		<description>Number of hosts connected to the device</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="keyfob-enable">
+		<item-type>Switch</item-type>
+		<label>Keyfob Enabled</label>
+		<description>Activates / deactivates the keyfob</description>
+		<category>Switch</category>
+		<config-description>
+			<parameter name="slot" type="integer"/>
+			<parameter name="signal" type="integer"/>
+		</config-description>
+	</channel-type>
+
+	<channel-type id="basic-shutter">
+		<item-type>Rollershutter</item-type>
+		<label>Shutter</label>
+		<description>Shutter command</description>
+		<category>Blinds</category>
+		<config-description>
+			<parameter name="up" type="integer"/>
+			<parameter name="down" type="integer"/>
+			<parameter name="stop" type="integer"/>
+			<parameter name="signal" type="integer"/>
+		</config-description>
+	</channel-type>
+
+	<channel-type id="shutter">
+		<item-type>Rollershutter</item-type>
+		<label>Shutter Position</label>
+		<description>Read / Write position of the shutter</description>
+		<state pattern="%d %unit%"/>
+		<config-description>
+			<parameter name="slot" type="integer"/>
+			<parameter name="signal" type="integer"/>
+			<parameter name="stop" type="integer"/>
+		</config-description>
+	</channel-type>
+
+	<channel-type id="alarm-timeout">
+		<item-type>Number:Time</item-type>
+		<label>Alarm Duration</label>
+		<category>zone_temporisee</category>
+		<state pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="alarm-volume">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Alarm Volume</label>
+		<category>sirene</category>
```

Comment:
```suggestion
		<category>Siren</category>
```

Reply:
Changed to `<category>oh:freeboxos:sirene</category>`